src/multi_vector_simulator/D1_model_components.py

```diff
@@ -430,11 +425,11 @@ def transformer_constant_efficiency_optimize(model, dict_asset, **kwargs):
                     variable_costs=dict_asset[DISPATCH_PRICE][VALUE],
                 )
             }
-            efficiencies = {}
-            for i in range(len(dict_asset[EFFICIENCY][VALUE])):
-                efficiencies[
-                    kwargs[OEMOF_BUSSES][dict_asset[INPUT_BUS_NAME][i]]
-                ] = 1 / (dict_asset[EFFICIENCY][VALUE][i])
+            efficiencies = {
+                kwargs[OEMOF_BUSSES][dict_asset[OUTPUT_BUS_NAME]]: dict_asset[
+                    EFFICIENCY
+                ][VALUE]
+            }
```

Comment:
Did you get why in the previous version we used `1/efficiency`? I guess because we need to get at it from the other way around if we use the inputs, dont we?

Reply:
Sounds reasonable.. but I don't know whether it was right before. I've checked the time series of a solar inverter (optimized capacity and efficiency = 0.9) and the time series are fine: reduces values by factor 0.9